<!-- -->

<launch>

    <node name="gotowaypoint_server" pkg="iris_test" type="gotowaypoint_server.py"  output="screen">
    </node>

    <node name="set_waypoints_node" pkg="set_waypoints" type="set_waypoints"  output="screen">
	<remap from="waypoint_info_topic"  to="/set_waypoints_info"/>
	<remap from="drone_info_topic"  to="/pose"/>
	<remap from="gotowaypoint_server"  to="/gotowaypoint"/>
	<param name="range" value="10.0"/>
	<param name="flight_time" value="0.0"/>
	<param name="emergency" value="false"/>
	<param name="stop_mission" value="false"/>
	<param name="minumum_radius" value="0.2"/>
	<rosparam>central_point:  [714757, 98200]</rosparam>  <!-- [x, y] -->
	<rosparam>final_position:  [714757, 98200]</rosparam>  <!-- [x, y] -->
	<rosparam>emergency_position:  [714757, 98200]</rosparam>  <!-- [x, y] -->
    </node>

</launch>
